Assets/Scripts/ArrowTriggeredIndicator.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Renderer))]
+public class ArrowTriggeredIndicator : MonoBehaviour
+{
+
+    private void Start()
+    {
+        GetComponent<Renderer>().material.color = Color.red;
```

Comment:
Since You already Get the material, why don't you cache it so you don't have to do another GetComponent call